health-sdk/sdk/src/main/res/layout/ghs_item_payment_provider_app.xml

```diff
@@ -1,23 +1,17 @@
 <?xml version="1.0" encoding="utf-8"?>
 <androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:app="http://schemas.android.com/apk/res-auto"
-    xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
-    android:layout_height="wrap_content">
+    android:layout_height="wrap_content"
+    android:layout_marginBottom="@dimen/ghs_medium"
+    android:background="@drawable/ghs_selectable_payment_provider_background"
+    >
 
-    <com.google.android.material.button.MaterialButton
-        android:id="@+id/ghs_payment_provider_app"
-        style="@style/GiniHealthTheme.Widget.Button.OutlinedButton.ListItem"
-        android:layout_width="0dp"
+    <include layout="@layout/ghs_payment_provider_selector_button"
+        android:id="@+id/ghs_selector_layout"
+        android:layout_width="match_parent"
```

Comment:
I would recommend not using match_parent inside a contraintlayout. it does not behave normally. Also, this layout would work perfectly fine with a FrameLayout :-)

Reply:
Switched to `FrameLayout`, thank you for your suggestion.